synapse_net/tools/distance_measure_widget.py

```diff
@@ -87,7 +87,8 @@ def on_measure_seg_to_object(self):
             seg_ids=seg_ids,
         )
         table_data = self._to_table_data(distances, seg_ids, endpoints1, endpoints2)
-        self._add_lines_and_table(lines, table_data, name="distances")
+        structure_layer_name = self._get_layer_selector_layer(self.image_selector_name2).name
+        self._add_lines_and_table(lines, table_data, name="distances-to-" + structure_layer_name)
```

Comment:
Let's shorten the `structure_layer_name` in the segmentation widget to avoid too long names here, see general comment.